docs/docs/configuration.md

```diff
@@ -104,6 +104,7 @@ sidebar_position: 3
 | prefixes | []string |  |  | Prefixes to accept |
 | announce-default | bool | false |  | Should a default route be exported to this peer? |
 | announce-originated | bool | true |  | Should locally originated routes be announced to this peer? |
+| announce-all | bool | false |  | Should all routes be exported to this peer? |
```

Comment:
Please use `go generate` to generate this documentation. Your description here and on `config.go#94` aren't the same. I think the description used here (in the md file) is better than the one used in `config.go#94`, so let's use that.